src/data/markdown/translated-guides/en/07 Misc/09 Glossary.md

```diff
@@ -1,189 +1,203 @@
+In k6 scripts, checks are a function that validates a response to a test request.
+<p>Read more: [Checks reference](/using-k6/checks)</p></dd>
 
-### Goja
+<dt id="duration">Duration</dt>
+<dd>The length of time that a test runs. When duration is set as an option, VU code runs for as many iterations as possible in the length of time specified (with possible finishing time given as a [graceful stop](#graceful-stop).
+<p>Read more: [Duration option reference](https://k6.io/docs/using-k6/k6-options/reference/#duration)</p></dd>
 
-**Goja** is a JavaScript runtime, purely written in go, that emphasizes standard compliance and performance. We use goja to allow for test scripting without having to compromise speed, efficiency or reliability, which would have been the case using NodeJS. For more details, see the [Goja repository on GitHub](https://github.com/dop251/goja).
+<dt id="dynamic-data">Dynamic data</dt>
+<dd>Data that might, or will, change between test runs. Common examples are order IDs, session tokens, or timestamps.
+<p>Read more: [Correlation and dynamic data example](https://k6.io/docs/examples/correlation-and-dynamic-data/)</p></dd>
 
-### Horizontal Scalability
+<dt id="endurance-testing">Endurance testing</dt>
+<dd>A synonym for [soak testing](#soak-test).</dd>
 
-**Horizontal Scalability** is a trait describing to what degree a system under test’s performance and capacity may be increased by adding more nodes (servers or computers for instance).
+<dt id="executor">Executor</dt>
+<dd>An property of a [scenario](#scenario) that configures VU behavior. You can use executors to configure whether to designate iterations as shared between VUs or to run per VU, or to configure or whether the VU concurrency is constant or changing</dd>
 
-### HTTP Archive
+<dt id="goja">Goja</dt>
+<dd>A JavaScript runtime, written in pure Go, that emphasizes standard compliance and performance. k6 uses Goja to enable test scripting without compromising speed, efficiency or reliability, compromises that would have been necessary with NodeJS
+<p>Read more: [Goja repository on GitHub](https://github.com/dop251/goja).</p></dd>
 
-An **HTTP Archive**, or **HAR file**, is a file containing logs of a browser interactions with the system under test. All of the included transactions are stored as JSON-formatted text. These archives may then be used to generate test scripts using, for instance, the [har-to-k6 Converter](https://github.com/k6io/har-to-k6). For more details, see the [HAR 1.2 Specification](http://www.softwareishard.com/blog/har-12-spec/).
+<dt id="graceful-stop">Graceful stop</dt>
+<dd>A period that lets VUs finish an iteration at the end of a load test. Graceful stops prevent abrupt halts in execution, and graceful ramp downs prevent unrealistic drops to zero VUs.
+<p>Read more: The [Graceful stop reference](https://k6.io/docs/using-k6/scenarios/graceful-stop/)</p></dd>
 
-### Iteration
+<dt id="horizontal-scalability">Horizontal scalability</dt>
```

Comment:
We don't talk much about horizontal scalability. Why do we need the term here? What are the conditions for a term to appear on the glossary?